### ISSUE TITLE
[Feature Request] torrent search

### ISSUE THREAD

**Issue by zero77** [novaroma-project/novaroma#5]
Can you add https://rarbg.to and https://torrentz.eu as another torrent search engine.

**Comment by toph24** [comment]
Yes! Especially now that Kickass Torrents is not currently available (possibly indefinite). Although it looks like this project may be on hiatus, since no commits in 7 months?

**Comment by emrahyumuk** [comment]
New version 0.9.9.3 is released. Critical Errors are fixed. Kickass and piratebay links are updated. 
We will add new torrent services and think about adding rarbg. Thanks.

**Comment by toph24** [comment]
Awesome! Thanks for the continued support.

On Aug 7, 2016, at 12:52 PM, Emrah Yumuk <notifications@github.com<mailto:notifications@github.com>> wrote:


New version 0.9.9.3 is released. Critical Errors are fixed. Kickass and piratebay links are updated.
We will add new torrent services and think about adding rarbg. Thanks.

-
